sale_commercial_partner/models/sale.py

```diff
@@ -8,5 +8,10 @@ class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
     commercial_partner_id = fields.Many2one(
-        'res.partner', related='partner_id.commercial_partner_id', store=True,
-        string='Commercial Entity', index=True)
+        'res.partner',
+        related='partner_id.commercial_partner_id',
+        store=True,
+        string='Commercial Entity',
+        index=True,
+        readonly=False
```

Comment:
Why do you put `readonly=False`? This can be considered a bug in previous versions.

Reply:
Related fields were editable earlier (till v11) so a user can always change that on Record, But from v12 related fields are readonly by default, So I added `readonly=False` to keep the same behaviour.

Reply:
As said, that's clearly a bug in previous versions. This mustn't be editable!

Reply:
Removed and Forced Pushed, Can be reviewed again.
(Do we need to do anything for the older versions ?)

Reply:
Yes, adding a PR for putting that as `readonly=True` will help all the <=11 installations with this module to avoid bottlenecks and possible DB locks.

Reply:
Will create PRs to fix this for v9 to 11. 
Do I need to create Separate PR for each or Are we doing same as odoo  (As they forward port Fixes to other versions )?

Reply:
In OCA we don't have a forward port process, as each version can require a lot of changes on each version. Sometimes, PSC cherry-picks the commit in upper versions in we find that it's equal. Making both PRs in this case can be the best option, and I can fast-track them.